admin_guide/upgrading_citus.rst

```diff
@@ -53,15 +53,15 @@ Step 1. Update Citus Package
 .. code-block:: bash
 
   sudo apt-get update
-  sudo apt-get install postgresql-10-citus-8.1
+  sudo apt-get install postgresql-10-citus-8.2
```

Comment:
We should update this to PG11? Seems like I missed this during the 8.1 changes, but these instructions for Citus version upgrades should assume the customer is on PG11.

Reply:
This is part of an example upgrading from 7.5 to 8.2. Since 7.x isn't compatible with pg 11, we don't attempt to upgrade both the Citus and pg versions at the same time.

We have a section later on for "Upgrading PostgreSQL version from 10 to 11" but I don't see where it tells people to install pg 11 in there, it kind of assumes that is already installed. Might be worthwhile for me to run through these instructions and actually try to upgrade from 7.5/pg10 -> 8.2/pg10 -> 8.2/pg11 and update whatever is missing.

Reply:
@sumedhpathak I updated the section to use pg11, and added a note about major upgrades. Could you take a look and see if that looks clear?